sdk/edger8r/linux/CodeGen.ml

```diff
@@ -209,8 +211,8 @@ let mk_ubridge_name (file_shortnm: string) (funcname: string) =
   sprintf "%s_%s" file_shortnm funcname
 
 let mk_ubridge_proto (file_shortnm: string) (funcname: string) =
-  sprintf "static sgx_status_t SGX_CDECL %s(void* %s)"
-          (mk_ubridge_name file_shortnm funcname) ms_ptr_name
+  sprintf "static sgx_status_t SGX_CDECL %s(sgx_enclave_id_t %s, void* %s)"
```

Comment:
Is this new feature optional? I wonder what Intel's policy on backwards incompatible changes is.